stdlib/CMakeLists.txt

```diff
@@ -38,6 +38,12 @@ else()
   set(SWIFT_STDLIB_STABLE_ABI_default FALSE)
 endif()
 
+if(SWIFT_BUILD_SDK_OVERLAY OR SWIFT_INCLUDE_TESTS)
+  set(SWIFT_BUILD_TEST_SUPPORT_MODULES_default TRUE)
+else()
+  set(SWIFT_BUILD_TEST_SUPPORT_MODULES_default FALSE)
+endif()
```

Comment:
Could you simplify this to something like:

```
set(SWIFT_BUILD_TEST_SUPPORT_MODULES_default
    (SWIFT_BUILD_SDK_OVERLAY OR SWIFT_INCLUDE_TESTS))
```

Reply:
I don't think CMake supports this syntax. And actually just looking over the existing CMake code we have, we seem to always be using the longer expanded variant, so let's keep it this way here, too.